manim/mobject/types/vectorized_mobject.py

```diff
@@ -833,14 +840,14 @@ def align_rgbas(self, vmobject):
                 setattr(self, attr, new_a1)
         return self
 
-    def get_point_mobject(self, center=None):
+    def get_point_mobject(self, center: Vector = None):
```

Comment:
In quotes because the class wasn't created up to this point. Could be wiser to move VectorizedPoint to the top and remove quotes, though.
```suggestion
    def get_point_mobject(self, center: Vector = None) -> "VectorizedPoint":
```